applications/GeoMechanicsApplication/custom_processes/apply_initial_uniform_stress_field.cpp

```diff
@@ -0,0 +1,44 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors: Richard Faasse
+//
+
+#include "apply_initial_uniform_stress_field.h"
+#include "includes/model_part.h"
+#include "utilities/parallel_utilities.h"
+
+namespace Kratos
+{
+
+ApplyInitialUniformStressField::ApplyInitialUniformStressField(ModelPart& rModelPart, const Parameters& rParameters)
+    : mrModelPart(rModelPart), mImposedStressVector(rParameters["value"].GetVector())
+{
+    block_for_each(mrModelPart.Elements(), [this](Element& rElement) {
```

Comment:
I'm wondering whether this could be a ref-to-const?
```suggestion
    block_for_each(mrModelPart.Elements(), [this](const Element& rElement) {
```

Reply:
good catch, made it `const auto&`